torch/csrc/jit/README.md

```diff
@@ -13,11 +12,66 @@ For concepts that are actual classes in the JIT, we use capitalized words, e.g.
 
 Sections start with a reference to the source file where the code related to the section resides.
 
-Core Program Representation
----------------------------
-
-
-### Modules ###
+## Table of Contents
```

Comment:
Are we going to manually updating this from now on? Or is there a way to auto-update

Reply:
Semi-manually is probably the least work, you can throw the whole doc into something like this [https://ecotrust-canada.github.io/markdown-toc/](https://ecotrust-canada.github.io/markdown-toc/) and get the TOC